<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddParticipantWindow</class>
 <widget class="QDialog" name="AddParticipantWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add participant</string>
  </property>
  <property name="styleSheet">
   <string>QDialog {
	background-color: rgb(201, 226, 252);
}
QScrollArea {
	border-bottom: 1px solid #b8c6d9;
	border-top: 1px solid #b8c6d9;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="topBar" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <widget class="QLineEdit" name="searchBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>24</height>
         </size>
        </property>
        <property name="styleSheet">
         <string>QLineEdit {
	border-image: url(:/gui/gfx/searchbox_inactive.png) repeat;
	border-width: 3px 21px 3px 12px;
}
QLineEdit:focus {
	border-image: url(:/gui/gfx/searchbox_active.png) repeat;
}</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="searchResultsStack">
     <property name="styleSheet">
      <string>QWidget#searchResults {
	background-color: #FFFFFF;
}
QWidget#searchResultsInvalid {
	background-color: #FFFFFF;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="searchResults">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QListView" name="searchResultsList">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="searchResultsInvalid">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="message">
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
